### PR TITLE
REFPLTB-2924: Required meta-dac-sdk-broadband kirkstone support for DAC

### PR DESCRIPTION
**Pull request by Ssandhyarani**
Reason for change: Adding Base code for meta-dac-sdk-broadband layer, fixes for build and iperf3 recipe bundle
Test Procedure: validate the dac build with meta-dac-sdk-broadband repo
Risks: Low

### CODE REVIEW

**Comment by mhughesacn** [comment]
Hi: I have cleared off most of the blackduck complaints.  Please will you append a credit to NOTICE as at least one recipe comes from there:
`Some material is derived from openembedded.org and is licensed in the same manner here (MIT for metadata; source code to match the associated recipe license).`

fdk-aac is commercial and being whitelisted.  Will the fdk-aac software being prepped here be run on a commercial product?  (Why: because any required commercial licenses will need to be in place.)

**Comment by Ssandhyarani** [comment]
Address Martin,
Please review

File: conf/layer.conf [hunk 1]
```diff
@@ -43,7 +36,7 @@ CMF_GIT_ROOT = "git://code.rdkcentral.com/r"
 CMF_GITHUB_MASTER_BRANCH ?= "master"
 CMF_GITHUB_ROOT ?= "git://github.com/rdkcentral"
 RDK_RELEASE ?= "1.99"
-LICENSE_FLAGS_WHITELIST += "commercial_fdk-aac"
+LICENSE_FLAGS_ACCEPTED += "commercial_fdk-aac"
```
Comment (cpokuru) [inline conf/layer.conf]: pls check this . we are accepting commercial.if aac not use pls remove this

**Review by cpokuru — COMMENTED** [review]
i believe this is generic and no changes done

File: NOTICE [hunk 2]
```diff
@@ -1,4 +1,6 @@
 
+Some material is derived from openembedded.org and is licensed in the same manner here (MIT for metadata; source code to match the associated recipe license).
```
Comment (cpokuru) [inline NOTICE:2]: do we need this

File: recipes-example/images/dac-image-iperf3.bb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+inherit  dac-image-iperf3
+
+IMAGE_INSTALL = "iperf3"
+
+# needed
+OCI_IMAGE_ENTRYPOINT = "/usr/bin/iperf3"
+APP_METADATA_PATH = "metadatas/iperf3-appmetadata.json"
+
+# optional
+OCI_IMAGE_AUTHOR = "LGi"
```
Comment (cpokuru) [inline recipes-example/images/dac-image-iperf3.bb:12]: pls use rdk specific rather than lgi

File: recipes-example/images/dac-image-iperf3.bb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+OCI_IMAGE_AUTHOR_EMAIL = "info@lgi.com"
```
Comment (cpokuru) [inline recipes-example/images/dac-image-iperf3.bb:13]: same here

File: conf/layer.conf (hunk 1 above)
Comment (Ssandhyarani) [reply conf/layer.conf]: Removed this 
https://github.com/rdkcentral/meta-dac-sdk-broadband/pull/1/commits/e579dcb376b6fb910146ddc0a09baf224ef25ea4#diff-22294624f20f807485d85b0f15eac2e1dac979796d77182b94b4896eb103c243

**Comment by Ssandhyarani** [comment]
for this comment > i believe this is generic and no changes done


Removed video related recipes layers and lines from conf due to below video related build issues and qt5.
1.
` ERROR: The following layer directories do not exist:
ERROR:    /home/jenkins/jenkinsroot/workspace/sep17/build/../meta-qt5
ERROR:    /home/jenkins/jenkinsroot/workspace/sep17/build/../meta-webkit
ERROR:    /home/jenkins/jenkinsroot/workspace/sep17/build/../meta-python2
ERROR:    /home/jenkins/jenkinsroot/workspace/sep17/build/../meta-rdk-video
ERROR:    /home/jenkins/jenkinsroot/workspace/sep17/build/../meta-rdk-cobalt
ERROR:    /home/jenkins/jenkinsroot/workspace/sep17/build/../meta-rdk-flutter`

2. 
`ERROR: ParseError at /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-example/example/qt-egl-test.bb:24: Could not inherit file classes/qmake5.bbclass
ERROR: Parsing halted due to errors, see error messages above`

3.
`ERROR: No recipes in default available for:
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-browser/wpewebkit/wpewebkit_%.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-containers/sloci-image/sloci-image-native_git.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-example/amazon-prime/amazon-prime-src_5.%.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-example/cobalt/libcobalt_23.lts.stable.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-example/cobalt/libcobalt_24.lts.stable.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-example/cobalt/libloader-app_23.lts.stable.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-example/cobalt/libloader-app_24.lts.stable.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-example/flutter-example/flutter-examples-slide-puzzle_git.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-example/netflix/netflix_5.3.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-example/netflix/rdk-gstreamer-utils.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-extended/wpe-framework/wpeframework-clientlibraries_2.1.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-extended/wpe-framework/wpeframework-clientlibraries_4.2.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-extended/wpe-framework/wpeframework-tools-native_2.1.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-extended/wpe-framework/wpeframework-tools-native_4.2.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-extended/wpe-framework/wpeframework_2.1.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-extended/wpe-framework/wpeframework_4.2.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-graphics/libsdl/libsdl2-ttf_2.0.15.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-graphics/libsdl2/libsdl2_2.0.12.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-dac-sdk-broadband/recipes-qt/qt/qtwayland_git.bbappend
  /home/jenkins/jenkinsroot/workspace/sep17--2/build/../meta-cmf/recipes-devtools/protobuf/protobuf_3.11.4.bbappend`

File: NOTICE (hunk 2 above)
Comment (mhughesacn) [reply NOTICE:2]: Hi Chandra: One recipe is matching in the blackduck tool, so is not original to here.  It is good practice to give some credit.  As the original recipe did not have a header, we add this generic note to NOTICE. - Martin

Comment (cpokuru) [reply NOTICE:2]: ok,thanks martin and sandhya

File: recipes-example/images/dac-image-iperf3.bb (hunk 3 above)
Comment (cpokuru) [reply recipes-example/images/dac-image-iperf3.bb]: once this is addressed we can move further

File: recipes-example/images/dac-image-iperf3.bb (hunk 4 above)
Comment (cpokuru) [reply recipes-example/images/dac-image-iperf3.bb]: once this is done we can move further

**Review by cpokuru — APPROVED** [review]

**Pull request merged**